### PR TITLE
Add `description` and `type` for `var.config_rules_paths`

### PR DESCRIPTION
**Pull request by MaxymVlasov**

### CODE REVIEW

File: .github/auto-release.yml [hunk 1]
```diff
@@ -17,7 +17,6 @@ version-resolver:
     - 'bugfix'
     - 'bug'
     - 'hotfix'
-    - 'no-release'
```
Comment (goruha) [inline .github/auto-release.yml:20]: @MaxymVlasov rollback this change please

File: .github/renovate.json [hunk 2]
```diff
@@ -4,9 +4,9 @@
     ":preserveSemverRanges"
   ],
   "labels": ["auto-update"],
+  "dependencyDashboardAutoclose": true,
```
Comment (goruha) [inline .github/renovate.json:7]: @MaxymVlasov rollback this changes

**Review by goruha — CHANGES_REQUESTED** [review]
see comments

**Comment by MaxymVlasov** [comment]
@goruha I can't, `Auto Format` introduces them automatically. I can try rebase to remove merge commit, but not sure is it helps

**Review by goruha — APPROVED** [review]

**Pull request merged**